pkg/kubelet/cm/dra/manager.go

```diff
@@ -85,8 +73,7 @@ func (m *ManagerImpl) PrepareResources(pod *v1.Pod) error {
 
 				if claimInfo := m.cache.get(claimName, pod.Namespace); claimInfo != nil {
 					// resource is already prepared, add pod UID to it
```

Comment:
i know it's not part of your change, but since we are in the process of updating this function, please capitalize this comment and move it above the if statement.